package.json

```diff
@@ -149,5 +149,9 @@
   "publishConfig": {
     "access": "public",
     "registry": "https://registry.npmjs.org/"
+  },
+  "dependencies": {
```

Comment:
I think we want everything on `devDependencies`, as I think `dependencies` end up in the npm package. Since the npm package is just delivering a built JS file, I don't think we want anything else there.